app/views/forms/_screen.html.erb

```diff
@@ -37,8 +37,101 @@
 
     </div>
 
-    <%# questions on the form %>
-    <%= index_table(Questioning, @form.questionings, :within_form => true) if form_mode != :new %>
+    <div class="questions">
```

Comment:
Perhaps `form-items-list`? They're not just questions, they're also groups. Also `questions` is such a common word I'd be worried about conflicts.